package.json

```diff
@@ -105,7 +105,7 @@
     "posttest": "cross-env MONGODB_VERSION=${MONGODB_VERSION:=4.0.4} MONGODB_TOPOLOGY=${MONGODB_TOPOLOGY:=standalone} MONGODB_STORAGE_ENGINE=${MONGODB_STORAGE_ENGINE:=mmapv1} mongodb-runner stop",
     "coverage": "cross-env MONGODB_VERSION=${MONGODB_VERSION:=4.0.4} MONGODB_TOPOLOGY=${MONGODB_TOPOLOGY:=standalone} MONGODB_STORAGE_ENGINE=${MONGODB_STORAGE_ENGINE:=mmapv1} TESTING=1 nyc jasmine",
     "start": "node ./bin/parse-server",
-    "prettier": "prettier --write '{src,spec}/{**/*,*}.js'",
+    "prettier": "prettier --write '{src,spec}/{**/*,*}.js' && npm run lint-fix",
```

Comment:
Not sure we should mix prettier with lint?

Reply:
it's a temporary fix, since our lint conflict with prettier, it's why we get at each PR some weird formatting changes.
We need to tune in another PR the Prettier/Linter config to avoid conflict and get a consistent formatting output !

Reply:
That's why I mean it should be a separate PR, or is this change required to be able to merge this PR?

Reply:
This has been discussed in https://github.com/parse-community/parse-server/pull/7172#issuecomment-775885168